src/d3d11/d3d11_interfaces.h

```diff
@@ -114,6 +142,33 @@ ID3D11VkExtDevice1 : public ID3D11VkExtDevice {
           uint32_t*               pCudaTextureHandle) = 0;
 };
 
+/**
+ * \brief Extended extended D3D11 device
+ * 
+ * Introduces methods to get virtual addresses and driver
+ * handles for resources, and create and destroy objects
+ * for D3D11-Cuda interop.
+ */
+MIDL_INTERFACE("f3112584-41f9-348d-a59b-00b7e1d285d6")
+ID3DLowLatencyDevice : public IUnknown {
```

Comment:
If it did, it would be much harder to [reuse the same interface in vkd3d-proton](https://github.com/HansKristian-Work/vkd3d-proton/pull/1739#discussion_r1373304094). This way we can query for only one interface in dxvk-nvapi without checking if we were passed D3D11 or D3D12 device.

But the comment above this inteface seems incorrect / copy-pasted from existing DeviceExt and should probably be changed.